### DISCUSSION TITLE
Plugin: add option to existing command?

### DISCUSSION THREAD

**Discussion by pawamoy** [pdm-project/pdm#578] [Q&A]
Is it possible to add an option to an existing command instead of registering a new command?
For example I would like to add a `--full` option to the `list` command, that would be mutually exclusive with `--graph` and `--freeze`. But I worry the mutually exclusive group will be difficult to reach to add arguments to it.

Maybe I can simply inherit from the command, and register in using the same name as to override it?
But then two plugins adding options to the same command would override each other (only one would win, the other would be discarded).

**Comment by frostming** [comment]
Yes, subclass and override with the same name.
